templates/pkg/crd_sdk.go.tpl

```diff
@@ -375,3 +353,19 @@ func (rm *resourceManager) newDeleteRequestPayload(
 	return res, nil
 }
 {{- end -}}
+
+{{ "" }}
```

Comment:
what's this?

Reply:
without this the generated .go files were not having newline between previous line `}` and `// setStatusDefaults ...` code that followed it.

Reply:
Ah, OK. You can modify the behaviour of the newline-trimming of the Go template by modifying the code directly above here from this:

```
{{- end -}}
```

to this:

```
{{- end }}
```

See here for a little more information on these "trim markers":

https://golang.org/pkg/text/template/#hdr-Text_and_spaces